tripal/tripal
# Adding Organism Image Field

# Tripal 4 Core Dev Task

### Issue #1637


### Tripal Version:  4.x

## Description


Adds an orgnanism image field to the organism content type.  This PR adds the field which uses the Drupal image field and also adds it to the default layout of the organism.  

## Testing?



- On a new tripal site:
   1.  Create an oraganism and add as many images as you like.  Upon saving, you should see  the images on the page.  
   2.  Go to `Tripal` > `Page Structure` and click on "manage display" under the "organism' content type.  Click the 'Apply Tripal Layout' button.   Now, return to the organism page and the images should be in a new details box named "Images"
- On an existing tripal site
   1.  Pull the code from this branch
   2. Go to `Tripal` > `Configuration` and click on 'Tripal Field Collection'. 
   3. Delete the collection named "Default Chado Fields"
   4. Clear the cache
   5. Re-import the field collection by going to `Tripal` > `Page Structure`  and click "Import Type Collection".  Check the box labeled "General Content Types (Chado)".  Run the job. You should now have an organism image field.  
   6. Create or edit an oraganism and add as many images as you like.  Upon saving, you should see  the images on the page.  
   2.  Go to `Tripal` > `Page Structure` and click on "manage display" under the "organism' content type.  Click the 'Apply Tripal
